scripts/gha_bump_packages.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)
+
+. "$SCRIPT_DIR/gha_common.sh"
+
+# bump versions of packages
+npx lerna version --yes --no-push --conventional-commits -m $PACKAGE_BUMP_COMMIT_MSG
```

Comment:
question: is the working directory the directory from where the script was triggered or does this script need to `cd ..` to get up to the project dir?

Reply:
Good question, [this](https://github.com/spinnaker/deck/blob/55fa5d00beb576d9d9215923bcc4b7ead64a4fba/scripts/gha_tag_bumped_packages.sh#L4) would suggest it's from the scripts directory.

Reply:
Doesn't hurt to navigate to the directory 👍 I'll add it in